agent/inner.py

```
@@ -128,9 +128,10 @@ def cell_exchange(self, message):
         update = self.simulation.generate_inner_update()
         division = update.get('division', [])
 
-        for daughter in division:
-            if not 'id' in daughter:
-                daughter['id'] = str(uuid.uuid1())
+        # TODO (Eran) -- is this ever needed? has potential to set new ids untracked by simulation
+        # for daughter in division:
+        # 	if not 'id' in daughter:
+        # 		daughter['id'] = str(uuid.uuid4())
```

Yes I see the value in leaving the failure states to trigger errors directly rather than provide default values that obscure the problem. I agree with you here.